docs/cli/environment.md

```diff
@@ -0,0 +1,131 @@
+---
+description: The `environment` command
+---
+
+# `environment`
+
+The `environment` command is used to manage environments in Keyshade. An environment is a context in which secrets are stored and accessed. It has a set of subcommands:
```

Comment:
~context in which secrets are stored and accessed~
context in which secrets and variables are stored and accessed

Reply:
will make the changes asap.